multicluster/apis/multicluster/v1alpha1/resourceimport_types.go

```diff
@@ -0,0 +1,96 @@
+/*
+Copyright 2021 Antrea Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1alpha1
+
+import (
+	v1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	mcs "sigs.k8s.io/mcs-api/pkg/apis/v1alpha1"
+
+	"antrea.io/antrea/pkg/apis/crd/v1alpha2"
+)
+
+// EndpointsImport imports Endpoints.
+type EndpointsImport struct {
+	Subsets []v1.EndpointSubset `json:"subsets,omitempty"`
+}
+
+// ExternalEntityImport imports ExternalEntity.
+type ExternalEntityImport struct {
+	ExternalEntitySpec *v1alpha2.ExternalEntitySpec `json:"externalentityspec,omitempty"`
+}
+
+// RawImport imports opaque resources.
```

Comment:
Then this one should be changed to RawResourceImport to be consistent.

Reply:
done.